src/cloud-api-adaptor/pkg/userdata/provision.go

```diff
@@ -2,45 +2,52 @@ package userdata
 
 import (
 	"context"
+	"crypto/sha256"
+	"crypto/sha512"
+	"encoding/base64"
+	"encoding/hex"
 	"fmt"
 	"log"
 	"os"
 	"path/filepath"
+	"strings"
 	"time"
 
 	"github.com/avast/retry-go/v4"
 	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers/aws"
 	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers/azure"
 	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers/docker"
+	toml "github.com/pelletier/go-toml/v2"
 	"gopkg.in/yaml.v2"
 )
 
-var logger = log.New(log.Writer(), "[userdata/provision] ", log.LstdFlags|log.Lmsgprefix)
+const (
+	ConfigParent = "/run/peerpod"
+	DigestPath   = "/run/peerpod/initdata.digest"
+	InitdataPath = "/run/peerpod/initdata"
```

Comment:
since coco initdata is specified to be a in toml format, do we want to call it `/run/peerpod/initdata.toml`?